client/src/components/History/HistoriesMenu.vue

```diff
@@ -24,32 +23,32 @@
             <b-dropdown-divider></b-dropdown-divider>
 
             <b-dropdown-item v-b-modal.history-selector-modal>
-                <Icon class="mr-1" icon="exchange-alt" />
+                <Icon fixed-width class="mr-1" icon="exchange-alt" />
                 <span v-localize>Change the Current History</span>
```

Comment:
```suggestion
                <span v-localize>Change History</span>
```

Reply:
Ahh, yeah, it might just be my bias knowing that there are actually implications for what is the 'current' or 'active' history elsewhere in the app that I like the explicit statement that that's changing, but users shouldn't have to consider this.  All they know is they're changing the history showing up in teh box here.